symfony/framework-bundle/3.3/src/Kernel.php

```diff
@@ -36,6 +36,7 @@ public function registerBundles()
 
     protected function configureContainer(ContainerBuilder $container, LoaderInterface $loader)
     {
+        // Feel free to remove the first parameter if you are using Symfony 4.0+ as it's the default behavior
```

Comment:
I think @stof was more recommending something like `Feel free to remove the "container.autowiring.strict_mode" parameter (...)` instead of replacing `1st` by `first` :smile:

Reply:
Hahaha ! Thank you @Pierstoval :-)